tests/platform_helper/test_command_copilot.py

```diff
@@ -80,6 +82,217 @@
+        new=Mock(return_value="foo"),
+    )
+    @patch("dbt_platform_helper.utils.application.get_aws_session_or_abort")
+    @patch("dbt_platform_helper.commands.copilot.get_aws_session_or_abort")
+    @mock_sts
+    @mock_s3
+    @mock_iam
+    @mock_kms
+    def test_s3_kms_arn_is_rendered_in_template(
+        self, mock_get_session, mock_get_session2, fakefs, kms_key_exists, kms_key_arn
+    ):
+        client = mock_aws_client(mock_get_session)
+        mock_aws_client(mock_get_session2, client)
+
+        client.get_parameters_by_path.return_value = {
+            "Parameters": [
+                {
+                    "Name": "/copilot/applications/test-app/environments/development",
+                    "Type": "SecureString",
+                    "Value": json.dumps(
+                        {
+                            "name": "development",
+                            "accountID": "000000000000",
+                        }
+                    ),
+                }
+            ]
+        }
+
+        if kms_key_exists:
+            client.describe_key.return_value = {"KeyMetadata": {"Arn": kms_key_arn}}
+        else:
+            client.exceptions.NotFoundException = boto3.client("kms").exceptions.NotFoundException
+            client.describe_key.side_effect = boto3.client("kms").exceptions.NotFoundException(
+                error_response={}, operation_name="describe_key"
+            )
+
+        fakefs.create_dir("./terraform")
+        fakefs.add_real_file(FIXTURES_DIR / "valid_workspace.yml", False, "copilot/.workspace")
+
+        create_test_manifests([S3_STORAGE_CONTENTS], fakefs)
+
+        CliRunner().invoke(copilot, ["make-addons"])
+
+        s3_addon = yaml.safe_load(Path(f"/copilot/web/addons/s3.yml").read_text())
+
+        assert (
+            s3_addon["Mappings"]["s3EnvironmentConfigMap"]["development"]["KmsKeyArn"]
+            == kms_key_arn
+        )
+
+    @pytest.mark.parametrize(
+        "addon_file, expected_service_addons",
+        [
+            (
+                "s3_addons.yml",
+                [
+                    "appconfig-ipfilter.yml",
+                    "subscription-filter.yml",
+                    "my-s3-bucket.yml",
+                    "my-s3-bucket-with-an-object.yml",
+                    "my-s3-bucket-bucket-access.yml",
+                ],
+            ),
+            (
+                "opensearch_addons.yml",
+                ["appconfig-ipfilter.yml", "subscription-filter.yml"],
+            ),
+            (
+                "rds_addons.yml",
+                ["appconfig-ipfilter.yml", "subscription-filter.yml"],
+            ),
+            (
+                "redis_addons.yml",
+                ["appconfig-ipfilter.yml", "subscription-filter.yml"],
+            ),
+            (
+                "aurora_addons.yml",
+                ["appconfig-ipfilter.yml", "subscription-filter.yml"],
+            ),
```

Comment:
Should we have Aurora stuff in the Terraform version?

Reply:
That's a good point. I'll remove it.